tools/RAiDER/utilFcns.py

```diff
@@ -401,20 +405,35 @@ def Chunk(iterable, n):
 
 def makeDelayFileNames(time, los, outformat, weather_model_name, out):
     '''
-    return names for the wet and hydrostatic delays
+    return names for the wet and hydrostatic delays.
+
+    # Examples:
+    >>> makeDelayFileNames(time(0, 0, 0), None, "h5", "model_name", "some_dir")
+    ('some_dir/model_name_wet_00_00_00_ztd.h5', 'some_dir/model_name_hydro_00_00_00_ztd.h5')
+    >>> makeDelayFileNames(None, None, "h5", "model_name", "some_dir")
+    ('some_dir/model_name_wet_ztd.h5', 'some_dir/model_name_hydro_ztd.h5')
     '''
-    str1 = time.isoformat() + "_" if time is not None else ""
-    str2 = "z" if los is None else "s"
-    str3 = 'td.{}'.format(outformat)
+    format_string = "{model_name}_{{}}_{time}{los}td.{ext}".format(
+        model_name=weather_model_name,
+        time=time.strftime("%H_%M_%S_") if time is not None else "",
+        los="z" if los is None else "s",
+        ext=outformat
+    )
     hydroname, wetname = (
-        '{}_{}_'.format(weather_model_name, dtyp) + str1 + str2 + str3
-        for dtyp in ('hydro', 'wet'))
+        format_string.format(dtyp) for dtyp in ('hydro', 'wet')
+    )
 
     hydro_file_name = os.path.join(out, hydroname)
     wet_file_name = os.path.join(out, wetname)
     return wet_file_name, hydro_file_name
 
 
+def make_weather_model_filename(name, time, ll_bounds):
+    return '{}_{}_{}N_{}N_{}E_{}E.h5'.format(
+        name, time.strftime("%Y-%m-%d %H_%M_%S"), *ll_bounds
```

Comment:
@Askaholic can you make this "%Y-%m-%dT%H_%M_%S"?

Reply:
Done